### PR TITLE
feat: PID issuing [SIW-313]

### PR DESCRIPTION
**Pull request by grausof**
Adds the integration towards the PID issuer for obtaining a PID credential.

#### List of Changes
- Added `PAR` request with JWT generator
- Added `auth token` request with DPoP generator
- Added `credential` request with proof on nonce generator

#### Motivation and Context
This PR ports the SDK made only for Android into typescript code

#### How Has This Been Tested?
Tested locally and through unit tests

#### How to use:
To understand how to get a PID refer to the `getPid` function within the sample app.

#### Checklist:
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sd-jwt/index.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const decode = <S extends z.AnyZodObject>(
     header: decodedJwt.protectedHeader,
     payload: decodedJwt.payload,
   });
+  console.log(rawDisclosures);
```
Comment (hevelius) [inline src/sd-jwt/index.ts:39]: Is this console.log wanted?

**Review by hevelius — CHANGES_REQUESTED** [review]

File: src/sd-jwt/index.ts (hunk 1 above)
Comment (hevelius) [reply src/sd-jwt/index.ts:39]: the example app is on /example/src instead this is on /src

Comment (grausof) [reply src/sd-jwt/index.ts]: Oops 😳
Fixed!

**Review by hevelius — APPROVED** [review]
LGTM

**Pull request merged**